<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserForm</class>
 <widget class="QWidget" name="UserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>625</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="guiIconsResources.qrc">
    <normaloff>:/icons/user-identity.png</normaloff>:/icons/user-identity.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="userGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Użytkownik</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="6">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="userLabel">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/images/images/Places-user-identity-icon.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="6">
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="1">
         <widget class="QLabel" name="idLabel">
          <property name="text">
           <string>--/--</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Typ karnetu</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="userTypeCombo"/>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="typeCombo"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="nameEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Imię</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="typeLabel">
          <property name="text">
           <string>Typ konta</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="surnameEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Nazwisko</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QCheckBox" name="promoCheckBox">
          <property name="text">
           <string>PROMOCJA</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="telefon">
          <property name="text">
           <string>Telefon</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="telefonEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="familyBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Rodzina</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="1" column="2">
       <widget class="QPushButton" name="familyAdd">
        <property name="text">
         <string>Dodaj</string>
        </property>
        <property name="icon">
         <iconset resource="guiIconsResources.qrc">
          <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="familyCombo"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="familiRm">
        <property name="text">
         <string>Usuń</string>
        </property>
        <property name="icon">
         <iconset resource="guiIconsResources.qrc">
          <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="5">
       <widget class="QListView" name="familyView">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>175</height>
        </size>
       </property>
       <property name="title">
        <string>Wejścia</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="1" column="1">
         <widget class="QRadioButton" name="radio30">
          <property name="text">
           <string>30 dni</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radio14">
          <property name="text">
           <string>14 dni</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTableView" name="entranceView">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="rfidGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>175</height>
        </size>
       </property>
       <property name="title">
        <string>Karta RFID</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableView" name="rfidView">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="rfidEditWidget" native="true">
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0" colspan="3">
            <layout class="QGridLayout" name="gridLayout_6">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <item row="0" column="2">
              <widget class="QToolButton" name="checkRFIDButton">
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="icon">
                <iconset resource="guiIconsResources.qrc">
                 <normaloff>:/icons/help-about.png</normaloff>:/icons/help-about.png</iconset>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="rfidEdit"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>RFID:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QPushButton" name="grabRFIDButton">
             <property name="text">
              <string>Przechwyć</string>
             </property>
             <property name="icon">
              <iconset resource="guiIconsResources.qrc">
               <normaloff>:/icons/Industry-Rfid-Tag-icon.png</normaloff>:/icons/Industry-Rfid-Tag-icon.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QPushButton" name="addRFIDButton">
             <property name="text">
              <string>Dodaj</string>
             </property>
             <property name="icon">
              <iconset resource="guiIconsResources.qrc">
               <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="3">
            <widget class="QPushButton" name="rmRFIDButton">
             <property name="text">
              <string>Usuń</string>
             </property>
             <property name="icon">
              <iconset resource="guiIconsResources.qrc">
               <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="extendBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>175</height>
        </size>
       </property>
       <property name="title">
        <string>Przedłuż</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Do:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QDateEdit" name="toEdit">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="3" column="1">
           <widget class="QPushButton" name="button6m">
            <property name="text">
             <string>6 miesięcy</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="button1m">
            <property name="text">
             <string>1 miesiąc</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="button14d">
            <property name="text">
             <string>14 dni</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="button3m">
            <property name="text">
             <string>3 miesiące</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="paymentBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="title">
        <string>Płatności</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item row="1" column="2">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="rmPaymentButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Usuń</string>
          </property>
          <property name="icon">
           <iconset resource="guiIconsResources.qrc">
            <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2" colspan="2">
         <widget class="QTableView" name="tableView">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="takePhotoButton">
       <property name="text">
        <string>Zrób zdjęcie</string>
       </property>
       <property name="icon">
        <iconset resource="guiIconsResources.qrc">
         <normaloff>:/icons/digikam.png</normaloff>:/icons/digikam.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="guiIconsResources.qrc">
         <normaloff>:/icons/dialog-ok.png</normaloff>:/icons/dialog-ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Dodaj użytkownika</string>
       </property>
       <property name="icon">
        <iconset resource="guiIconsResources.qrc">
         <normaloff>:/icons/contact-new.png</normaloff>:/icons/contact-new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addKeyButton">
       <property name="text">
        <string>Dodaj klucz</string>
       </property>
       <property name="icon">
        <iconset resource="guiIconsResources.qrc">
         <normaloff>:/icons/preferences-desktop-cryptography.png</normaloff>:/icons/preferences-desktop-cryptography.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>Edytuj</string>
       </property>
       <property name="icon">
        <iconset resource="guiIconsResources.qrc">
         <normaloff>:/icons/user-properties.png</normaloff>:/icons/user-properties.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Anuluj</string>
       </property>
       <property name="icon">
        <iconset resource="guiIconsResources.qrc">
         <normaloff>:/icons/dialog-cancel.png</normaloff>:/icons/dialog-cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="entranceButton">
       <property name="text">
        <string>Wejście</string>
       </property>
       <property name="icon">
        <iconset resource="guiIconsResources.qrc">
         <normaloff>:/icons/view-task-add.png</normaloff>:/icons/view-task-add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="extendButton">
       <property name="text">
        <string>Przedłuż</string>
       </property>
       <property name="icon">
        <iconset resource="guiIconsResources.qrc">
         <normaloff>:/icons/view-income-categories.png</normaloff>:/icons/view-income-categories.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radio14</tabstop>
  <tabstop>radio30</tabstop>
  <tabstop>rfidEdit</tabstop>
  <tabstop>checkRFIDButton</tabstop>
  <tabstop>grabRFIDButton</tabstop>
  <tabstop>addRFIDButton</tabstop>
  <tabstop>rmRFIDButton</tabstop>
  <tabstop>toEdit</tabstop>
  <tabstop>button14d</tabstop>
  <tabstop>button1m</tabstop>
  <tabstop>button3m</tabstop>
  <tabstop>button6m</tabstop>
  <tabstop>takePhotoButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>entranceButton</tabstop>
  <tabstop>extendButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
  <include location="guiIconsResources.qrc"/>
 </resources>
 <connections/>
</ui>
